src/service/position-observer-impl.js

```diff
@@ -0,0 +1,402 @@
+import {AmpContext} from '../../3p/ampcontext';
```

Comment:
Please don't review this file in this PR. The changes in this file are to be merged with #8940 and this PR will be dependent on #8940